### PR TITLE
Add blog-post for week 16

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Signed-off-by: Frantisek Lachman <flachman@redhat.com>

### CODE REVIEW

File: content/posts/April-2021.md [hunk 1]
```diff
@@ -28,3 +28,18 @@ weight: 76
   ([packit-service#1046](https://github.com/packit/packit-service/pull/1046)).
   This way users will receive a more immediate feedback about the Service
   handling their requests.
+
+## Week 16 (April 19th - April 23th)
+
+- The `current_version_command` and `create_tarball_command` config options are being deprecated
+  in favour of [actions](https://packit.dev/docs/actions/).
+  The issue will be created in the affected repositories once we find those options in use.
```
Comment (jpopelka) [inline content/posts/April-2021.md:36]: - `The` suggests some specific issue
- `once` suggests we are actively searching the repositories for such options, which is not the case, is it?

```suggestion
  An issue will be created in the affected repositories if we find those options in use.
```

File: content/posts/April-2021.md [hunk 2]
```diff
@@ -28,3 +28,18 @@ weight: 76
...
+  ([packit-service#1064](https://github.com/packit/packit-service/pull/1064))
+- The result pages have been replaced by the views on our dashboard.
+  Let us know what do you think about that and what information do you want to see there.
+  You can expect more changes on this field.
+  - The result views have been implemented by [@IceWreck](https://github.com/IceWreck)
+    ([dashboard#73](https://github.com/packit/dashboard/pull/73)).
+  - The integration on packit-service side has been done by [Maťo](https://github.com/mfocko)
+    ([packit-service#1056](https://github.com/packit/packit-service/pull/1056),
+    ([dashboard#95](https://github.com/packit/dashboard/pull/95)).
```
Comment (jpopelka) [inline content/posts/April-2021.md:45]:
```suggestion
    [dashboard#95](https://github.com/packit/dashboard/pull/95)).
```

**Review by jpopelka — APPROVED** [review]

File: content/posts/April-2021.md (hunk 1 above)
Comment (lachmanfrantisek) [reply content/posts/April-2021.md:36]: You are right. We are not actively searching. We create an issue during the processing of some event.

**Review by TomasTomecek — APPROVED** [review]
LGTM

**Pull request merged**